paasta_tools/utils.py

```diff
@@ -1657,3 +1655,20 @@ def mean(iterable):
     Returns the average value of an iterable
     """
     return sum(iterable) / len(iterable)
+
+
+class NoTtyError(Exception):
+    pass
+
+
+def prompt_pick_one(sequence):
+    if not sys.stdin.isatty():
+        raise NoTtyError
```

Comment:
Right now everything that's using this method catches the `NoTtyError` and prints something helpful. Returning None is what happens when the user quits the prompt by typing 'Quit'.